workshop/permissions.py

```diff
@@ -23,8 +21,6 @@ class AllowWorkshopHeadOrContact(permissions.BasePermission):
     message = "You are not authorized to perform this task"
 
     def has_object_permission(self, request, view, obj):
-        if request.method in permissions.SAFE_METHODS:
-            return True
```

Comment:
Removing this will lead to `401 Unauthorized error` in `/workshops/{id}/` `GET` request, if the user isn't a Workshop Head or Contact. Haven't you tested this?

I guess SAFE_METHODS are ('GET', 'HEAD', 'OPTIONS'), so adding these lines won't be a problem.
However, removing this line in `AllowWorkshopHead` class can be considered, but also check if it doesn't break anything.